.changeset/blue-ducks-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+'@moonbeam-network/xcm-config': major
```

Comment:
```suggestion
'@moonbeam-network/xcm-config': patch
```

it should be a `patch` version, not a `major`

The changeset prompt asks you about which packages require `major` or `minor` bumps, if none apply it will define it as `patch`

https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md#i-am-in-a-multi-package-repository-a-mono-repo

We follow https://semver.org/